standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -5567,10 +5570,16 @@ private void removeTxnsFromMinHistoryLevel(Connection dbConn, List<Long> txnids)
     }
   }
 
-  private static synchronized DataSource setupJdbcConnectionPool(Configuration conf, int maxPoolSize, long getConnectionTimeoutMs) throws SQLException {
+  private static synchronized DataSource setupJdbcConnectionPool(Configuration conf, int maxPoolSize, long getConnectionTimeoutMs) {
```

Comment:
I don't think it needs to be static and synchronized. The only place where it's used is setConf method under the synchronized block

Reply:
Done